schemas/codify.xsd

```diff
@@ -1,50 +1,92 @@
 <?xml version="1.0" encoding="utf-8"?>
-<xs:schema xmlns="https://code.dccouncil.us/schemas/codify"
-           xmlns:xs="http://www.w3.org/2001/XMLSchema"
-           elementFormDefault="qualified"
-           targetNamespace="https://code.dccouncil.us/schemas/codify">
-  <xs:element name="annotation" type="xs:anyType"/>
-  <xs:element name="applicability" type="xs:anyType"/>
-  <xs:element name="expire" type="xs:anyType"/>
-  <xs:element name="create-sub-container" type="xs:anyType"/>
-  <xs:element name="find-replace" type="xs:anyType"/>
-  <xs:element name="funded-anno" type="xs:anyType"/>
-  <xs:element name="ignore" type="xs:anyType"/>
-  <xs:element name="insert" type="xs:anyType"/>
-  <xs:element name="insert-code-container" type="xs:anyType"/>
-  <xs:element name="not-funded-anno" type="xs:anyType"/>
-  <xs:element name="organic" type="xs:anyType"/>
-  <xs:element name="redesignate" type="xs:anyType"/>
-  <xs:element name="redesignate-para" type="xs:anyType"/>
-  <xs:element name="recodify-section" type="xs:anyType"/>
-  <xs:element name="remove" type="xs:anyType"/>
-  <xs:element name="repeal" type="xs:anyType"/>
-  <xs:element name="replace" type="xs:anyType"/>
-  <xs:element name="street-designation-anno" type="xs:anyType"/>
-  <xs:element name="temporary" type="xs:anyType"/>
-  <xs:element name="any">
-    <xs:complexType>
-      <xs:sequence maxOccurs="unbounded">
-        <xs:any processContents="lax"/>
-      </xs:sequence>
+<xs:schema
+    xmlns="https://code.dccouncil.us/schemas/codify"
+    xmlns:lib="https://code.dccouncil.us/schemas/dc-library"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" targetNamespace="https://code.dccouncil.us/schemas/codify">
+    <xs:import namespace="https://code.dccouncil.us/schemas/dc-library" schemaLocation="dc-library.xsd"/>
+    <xs:element name="annotation" type="anyType"/>
+    <xs:element name="applicability" type="anyType"/>
+    <xs:element name="expire" type="anyType"/>
+    <xs:element name="create-sub-container" type="anyType"/>
+    <xs:element name="find-replace" type="anyType"/>
+    <xs:element name="funded-anno" type="anyType"/>
+    <xs:element name="ignore" type="anyType"/>
+    <xs:element name="insert">
+        <xs:complexType>
+            <xs:complexContent>
+                <xs:extension base="insertType">
+                    <xs:attributeGroup ref="lib:targetAttributes"/>
+                    <xs:attributeGroup ref="targetAttributes"/>
+                    <xs:attribute name="after" type="xs:string" use="optional"/>
+                    <xs:attribute name="before" type="xs:string" use="optional"/>
+                    <xs:attribute name="history" type="xs:boolean" use="optional"/>
+                    <xs:attribute name="history-prefix" type="xs:string" use="optional"/>
+                    <xs:attribute name="num-value" type="xs:string" use="optional"/>
+                    <xs:attribute name="technical" type="xs:string" use="optional"/>
```

Comment:
The `technical` attribute here is used for more than just "true". For example, `technical="this should be (B)(iii)"` (see `dc\council\periods\21\laws\21-140.xml:228:0`).

If I remember correctly, everywhere else it is simply `technical="true"`.

It seems like _all_ `technical` attributes should be `xs:string`, as opposed to some `xs:string` and some `xs:boolean` or similar variation.

Is this correct?